### DISCUSSION TITLE
Any news on Syncfusion releasing components for Uno ?

### DISCUSSION THREAD

**Discussion by DomagojJugovich** [unoplatform/uno#11298] [Q&A] [answered]
Or other well know third party provider ...

**Comment by sasakrsmanovic** [comment, marked as answer]
Hello Domagoj, 

Thanks for reaching out. What components in particular are you looking for and for what targets. We believe that between WinUI controls + Windows Community Toolkit + Uno Toolkit you have a lot of choice for UI controls. 

In addition, LiveCharts and LightningChart have full support for Uno Platform https://github.com/beto-rodriguez/LiveCharts2/releases  and https://platform.uno/blog/lightningchart-introduces-uno-platform-support/ 

I don't have any news to share regarding Syncfusion; best to contact them directly. Also, you can consider the beta of Infragistics controls https://www.infragistics.com/products/uno-platform , or for maps you have beta of MAPS UI https://mapsui.com/documentation/getting-started-uno.html 

I hope that helps. But again, I'd be very interested to hear what controls are you missing, and for which target platforms. 

Sasha

**Reply by DomagojJugovich** [reply]
I saw all that issues (https://github.com/unoplatform/uno/issues/7614), announcements for syncfusion support a while ago, and see that a lot of serious players (Syncfusion, Devexpress, Telerik) have good  support for Blazor. And uno is left out.
And don't get me wrong I am rooting for UNO.

Syncfusion beta demo is unusable now, also Infragistics don't work.

**I asked here for new because I thought that you have some communication with them (Syncfusion).**
I saw a thread on Syncfusion site that they dont have any plans for Uno for now.
This is all you can find :
      https://www.syncfusion.com/forums/176644/syncfusion-uno-platform-support
      https://www.syncfusion.com/feedback/20574/support-for-uno-platform
I also commented there about UNO support.

I would really like to use UNO on WASM, and other platforms of course , but I don't like JS and hot it is so easy bug prone (and I worked with it for 20 years, also C# and Java a lot in that 20 years). Only other choice is Blazor (C# + HTML), but it only for web projects.

It is a dream to work for web in C# and XAML.

In my firm we did rewrite small Inventory app on android with UNO a year ago, with just plain UNO without any third party components.

For any serious consideration we would need third party support, hopefully Syncfusion because we are already a customer.

I played with UNO on multiple occasions in last 2 years , for last I created an issue even :
https://github.com/unoplatform/uno/issues/11184

Thx, and keep the good work.

**Comment by FrozDark** [comment]
https://github.com/unoplatform/uno/issues/7614
